### ISSUE TITLE
Select box placeholder option not shown on dropdown list?

### ISSUE THREAD

**Issue by titusdecali** [wearebraid/vue-formulate#439]
**Describe the bug**
I used this package on a previous project and type="select" always showed the `placeholder` as the first option on the dropdown list of the select box. I tried the same for a different project and it now shows the first option as selected (although nothing has yet been selected) and the placeholder is not shown at all.

**To Reproduce**
Although the docs say that the placeholder will be shown, the example also doesn't show the placeholder among the options. If you inspect the element, the placeholder is set as `hidden`
https://vueformulate.com/guide/inputs/types/select/#select-2

![CleanShot 2021-06-06 at 16 18 36@2x](https://user-images.githubusercontent.com/4532534/120916162-e877b580-c6e2-11eb-877d-68bf6992c4ed.jpg)

**Expected behavior**
I think choosing to show or hide the placeholder within the dropdown list options should be controlled via a prop on the `formulate-input`

**Screenshots**
Current project:
![CleanShot 2021-06-06 at 16 05 29@2x](https://user-images.githubusercontent.com/4532534/120915866-14923700-c6e1-11eb-9733-5023f745394b.jpg)

Previous project:
![CleanShot 2021-06-06 at 16 06 20@2x](https://user-images.githubusercontent.com/4532534/120915873-270c7080-c6e1-11eb-891d-b5bd7f57f33b.jpg)

**Comment by nsetyo** [comment]
+1 on this, for an optional form user should be able to deselect option and choosable placeholder would be workaround

**Comment by titusdecali** [comment]
> +1 on this, for an optional form user should be able to deselect option and choosable placeholder would be workaround

Yes, it would be nice to also have a prop for allowing this option to be selected as nsetyo mentioned.
